views/posts/index.hbs

```diff
@@ -25,28 +25,24 @@
     <div class="post-container card" id="{{this._id}}">
       <div class="actual-post">
         <div class="profile-picture">
-          <a href="/profile/{{this.user._id}}">
```

Comment:
is this not taking away the link to the person's profile?

Reply:
Good spot